### PR TITLE
minor changes to wording

### PR DESCRIPTION
**Pull request by techieari**

### CODE REVIEW

**Comment by PRMerger15** [comment]
@arianapadilla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@arianapadilla - can you resolve the merge conflicts?  After that, @Karl-Bridge-Microsoft can review the changes.

File: hub/apps/design/globalizing/use-mat.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ Before you start the Microsoft Store Certification process, you must exclude the
 
 ## Telemetry
 
-The Multilingual Application Toolkit Visual Studio (MAT VS) extension collects anonymous usage data and sends it to Microsoft servers to help improve our products and services. Telemetry is off by default and can be enabled or disabled at any time. 
+The Multilingual App Toolkit Visual Studio (MAT VS) extension collects anonymous usage data and sends it to Microsoft servers to help improve our products and services. Telemetry is off by default and can be enabled or disabled at any time. For more details, see [Multilingual App Toolkit 4.0 FAQ & troubleshooting](https://docs.microsoft.com/en-us/windows/apps/design/globalizing/mat-faq-troubleshooting).
```
Comment (Jay-o-Way) [inline hub/apps/design/globalizing/use-mat.md]:
```suggestion
The Multilingual App Toolkit Visual Studio (MAT VS) extension collects anonymous usage data and sends it to Microsoft servers to help improve our products and services. Telemetry is off by default and can be enabled or disabled at any time. For more details, see [Multilingual App Toolkit 4.0 FAQ & troubleshooting](windows/apps/design/globalizing/mat-faq-troubleshooting).
```
Don't hardcode these the beginning or _lang_ codes.

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]

**Comment by PRMerger10** [comment]
@arianapadilla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by PRMerger12** [comment]
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Pull request merged**